core/edit/edit.talon

```diff
@@ -125,12 +125,18 @@ new line above: edit.line_insert_up()
 new line below | slap: edit.line_insert_down()
 
 # Insert padding with optional symbols
-(pad | padding): user.insert_between(" ", " ")
+(padding): user.insert_between(" ", " ")
 (pad | padding) <user.symbol_key>+:
     insert(" ")
     user.insert_many(symbol_key_list)
     insert(" ")
 
+# Insert delimiter pairs
+<user.delimiter_pair>: user.delimiter_pair_insert(delimiter_pair)
+
+# Wrap selection with delimiter pairs
+<user.delimiter_pair> that: user.delimiter_pair_wrap_selection(delimiter_pair)
```

Comment:
There is one subtlety with the new wrap command versus the deprecated commands: 
the cursor winds up within the delimiters when wrapping; previously, the cursor would wind up outside. I'm not entirely sure this change is undesirable. If the intent is to replace the deprecated commands, should we preserve the previous functionality?

Reply:
I suppose there are three scenarios we could cover with the grammar.

1. cursor between the delimiters. Done.
2. Wrapping selection with decimeters. Done, but need to settle the cursor position.
3. Patch in the delimiters, cursor after. I believe this was the original purpose of the "empty <delimiter>" commands, but they no longer behave as such. Should we bother with this?

Reply:
Personally I prefer the cursor ending up inside but we can change it if people prefer. Should we get a few additional voices on this?

Reply:
Yeah, let's see what @nriley and @phillco think :)